packages/design-tokens/src/scss/generated/_gradients.scss

```diff
@@ -1,8 +1,12 @@
 // Generated Gradients from Supernova. Do not edit manually.
+$basic-ltr: linear-gradient(90deg, #fff 0%, #ffffff00 100%) !default;
+$basic-rtl: linear-gradient(270deg, #fff 0%, #ffffff00 100%) !default;
 $gradient-diagonal: linear-gradient(135deg, #073256 0%, #185c95 100%) !default;
 $gradient-horizontal: linear-gradient(90deg, #073256 0%, #185c95 100%) !default;
```

Comment:
Just a question of uniformity: shouldn't `ltr` and `rtl` be also `horizontal`? Or, can `horizontal` be further (or instead) specified as `ltr`/`rtl`?

Reply:
You mean like `basic-horizontal-ltr/rtl`? I would understand that `horizontal` is a duplicit information to the `ltr`/`rtl`. 
We considered naming like `gradient-background-basic-horizontal-start/end`, but it's not such specific.

Reply:
With the JDS team we agreed on the renaming or maybe unifying existing gradients in the nearly future.
Maybe won't be an wrong idea replace abbreviations with the whole name.